picard/tagger.py

```diff
@@ -347,11 +351,16 @@ def pipe_server(self):
         while self.pipe_handler.pipe_running:
             messages = [x for x in self.pipe_handler.read_from_pipe() if x not in IGNORED]
             if messages:
+                print(messages)
```

Comment:
I'm guessing these print statements were for your local debugging during development and shoud be removed?

Reply:
Yeah, it’s all for debugging purposes. As you can see, QUIT doesn’t work and I haven’t implemented most of the commands yet. So it’ll remain here for a while.